erp-data-exchange-specification.md

```diff
@@ -154,63 +156,111 @@ The `utilisationPeriods.csv` describes utilisation periods.
+### checkIns.csv
+
+The `checkIns.csv` file describes the relation between a tenant, or company and a utilisation period.
+
+#### Fields
+
+| Field                   | Type                        | Description                                                                                           |
+| ----------------------- | --------------------------- | ----------------------------------------------------------------------------------------------------- |
+| **importType**          | [Import Type](#import-type) | Must always use "insert"
+| **utilisationPeriodId** | [UUID](#uuid-type)          | The foreign UUID of the utilisation period the tenant check-in is for (_utilisationPeriods.csv_ `id`) |
+| **residentId**          | [UUID](#uuid-type)          | The foreign UUID of the tenant or the company this check-in is for (_tenantss.csv_ or _companies.csv_ `id`)                           |
+| **registrationCodeId**          | [UUID](#uuid-type)          | The foreign UUID of the registrationCode this check-in is for this is optional. (_registrationCodes.csv_ `id`)                           |
+
+#### Example
+
+```csv
+importType,residentId,utilisationPeriodId, registrationCodeId
+insert,7d87a383-1778-401b-a421-b60c900479c3,db8b732f-e0ff-41d9-9c15-ca1be2776fd4,f598b428-997f-11e8-9eb6-529269fb1459
+insert,7d87a383-1778-401b-a421-b60c900479c3,db8b732f-e0ff-41d9-9c15-ca1be2776fd4,f598b6bc-997f-11e8-9eb6-529269fb145
+insert,7d87a383-1778-401b-a421-b60c900479c3,ad257d42-1078-4279-9918-e774859555ae,1be814ca-9980-11e8-9eb6-529269fb1459
+insert,7d87a383-1778-401b-a421-b60c900479c3,ad257d42-1078-4279-9918-e774859555ae,1be81286-9980-11e8-9eb6-529269fb1459
+```
+
+### registrationCodes.csv
+
+The `registrationCodes.csv` file describes registration codes for one or more utilizationPeriods. It contains the registrationCode and the reseidentId of those entities that are checked into these utilizationPeriods.
 
 #### Fields
 
 | Field                | Type                        | Description              |
 | -------------------- | --------------------------- | ------------------------ |
 | **importType**       | [Import Type](#import-type) | Must always use "insert"
-| **id**               | [UUID](#uuid-type)          | Your UUID for the tenant |
+| **id**               | [UUID](#uuid-type)          | UUID to identify registrationCode |
 | **registrationCode** | [string](#string-type)      |
 
 #### Example
 
 ```csv
-importType,id,registrationCode
+importType,id,residentId,registrationCode
 insert,db8b732f-e0ff-41d9-9c15-ca1be2776fd4,3fdfdssf7
 insert,ad257d42-1078-4279-9918-e774859555ae,18ggjbs9d
 insert,2f0fc4c6-3a5e-4cd7-9007-9add50653be5,ffhh3t4jg
 insert,41f93044-e050-4a33-9e52-ebdada55f9a7,1awh74hgf
 ```
 
-### tenantCheckIns.csv
+### tenants.csv
 
-The `tenantCheckIns.csv` file describes the relation between a tenant and a utilisation period.
+The `tenants.csv` file describes a tenant.
```

Comment:
Shouldn't we be consistent in terms of why is the file called `tenants.csv` when it contains residentIds?